pkg/labels/filter.go

```diff
@@ -34,13 +34,31 @@ type LabelPrefix struct {
 	Source string `json:"source"`
 }
 
+func ParseLabelPrefix(label string) *LabelPrefix {
+	labelPrefix := LabelPrefix{}
+	t := strings.SplitN(label, ":", 2)
+	if len(t) > 1 {
+		labelPrefix.Source = t[0]
```

Comment:
Then the label will not match. How do you know the valid label sources at this point. It could be anything. As we make the label retrieving more modular, we will have no idea about all possible sources of labels.